### ISSUE TITLE
Support for sqlAlchemy 2.0

### ISSUE THREAD

**Issue by Bensebabillal** [amisadmin/fastapi_amis_admin#76]
Hi guys, 

Con we add support for sql alchemy 2.0 please ?

**Comment by amisadmin** [comment]
Yes, sqlalchemy 2.0 support will begin after pydantic V2 is released.

**Comment by honglei** [comment]
pydantic V2 is not needed for supporting sqlalchemy2.0: 
[migration to sqlalchemy 2.0](https://github.com/tiangolo/sqlmodel/pull/563) 

admin.py:
insfield.prop.secondary -> insfield.property.secondary

_sqlmodel.py:
stmt.with_only_columns([self.pk])->stmt.with_only_columns(self.pk)

**Comment by amisadmin** [comment]
> pydantic V2 is not needed for supporting sqlalchemy2.0: [migration to sqlalchemy 2.0](https://github.com/tiangolo/sqlmodel/pull/563)
> 
> admin.py: insfield.prop.secondary -> insfield.property.secondary
> 
> _sqlmodel.py: stmt.with_only_columns([self.pk])->stmt.with_only_columns(self.pk)

Thank you for your feedback. I've been following the sqlmodel project.

**Comment by mikoda1995** [comment]
@amisadmin firstly thank you for this project. Do you have any plans to support sqlalchemy 2.0 now?

**Comment by peter65374** [comment]
it seems that sqlalchemy 2.0 was supported now. 👍

**Comment by mikoda1995** [comment]
> it seems that sqlalchemy 2.0 was supported now. 👍

yep, but seems fastapi-user-auth is still depends on sqlmodel and fastapi-amis-admin<0.6.0
